### DISCUSSION TITLE
关于 nameserver 和 domain-rules 指令

### DISCUSSION THREAD

**Discussion by ghost** [pymumu/smartdns#1420] [Q&A] [answered]
在实现指定域名使用特定服务器组解析的功能上，两者有区别么？

例如，定义服务器组 cloud 和域名集合 example：
```
server 1.1.1.1 -group cloud
domain-set -name example -file /etc/smartdns/example.conf
```
如果想指定 example 集合使用 cf 组解析，那么用以下两者有区别么？
```
nameserver /domain-set:example/cf
domain-rules /domain-set:example/ -nameserver cf
```

如果没区别，可否认为 domain-rules 已经包含了 nameserver 功能，而 nameserver 已经没有存在的意义了？

**Comment by pymumu** [comment, marked as answer]
是一样的。
domain-rules包含了其他类似的选项，比如address，ipset等。

其他选项存在的意义在于和dnsmasq写法类似，并且对于单个域名的配置，可以很方便。
